apps/src/templates/instructions/teacherFeedback/TeacherFeedback.jsx

```diff
@@ -137,14 +133,7 @@ export class TeacherFeedback extends Component {
       analytics_section_id: this.props.selectedSectionId
     };
 
-    $.ajax({
-      url: '/api/v1/teacher_feedbacks',
-      method: 'POST',
-      contentType: 'application/json;charset=UTF-8',
-      dataType: 'json',
-      data: JSON.stringify({teacher_feedback: payload}),
-      headers: {'X-CSRF-Token': this.props.token}
-    })
+    updateTeacherFeedback(payload, this.props.token)
```

Comment:
Sorry, I need to run to a bunch of meetings so I haven't had a chance to look at this as closely as I'd like, but I'm wondering something along the lines of: It looks like we get data from three different places in this component: props, redux, and call to API.  Why do we have all three and what's the organizing principle for determining where we get a particular piece of data from?

Reply:
I would expect data to come from:
- Redux: when it's set higher up in the tree/ needed across multiple components
- API: if we're at the root of the part of the tree that needs that data
- Props: it's not data that's necessary to store in redux but it gets set higher up the tree

Reply:
Got it, thanks for the explanation.  I'm hoping we can simplify this further as we think about our future architecture.